sysbuild/CMakeLists.txt

```diff
@@ -148,7 +153,7 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
   set(target_length)
 
   if(SB_CONFIG_BOOTLOADER_MCUBOOT)
-    if(SB_CONFIG_PARTITION_MANAGER)
+    if(SB_CONFIG_PARTITION_MANAGER OR SB_CONFIG_SECURE_BOOT)
```

Comment:
It might be a strange idea, but wouldn't it be easier if the B0 and MCUboot got merged together and be flashed separately from the app?